app/src/pipelines/generate_referrals_from_doc/pipeline_wrapper.py

```diff
@@ -37,7 +37,7 @@ def setup(self) -> None:
             "prompt_builder",
             ChatPromptBuilder(
                 template=haystack_utils.get_phoenix_prompt("generate_referrals"),
-                required_variables=["query", "supports", "resource_json"],
+                required_variables=["query", "supports", "response_json"],
```

Comment:
Renamed variable to (1) note that it is semantically different from prior versions and (2) make it more generic.